helm/adelphi/templates/prometheus-configmap.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-configmap
+  namespace: {{ .Values.namespace }}
+data:
+  prometheus.yml: |
+    # this directory was added by nosqlbench for docker-metrics
+    # it is used by the prometheus docker container
+    global:
+      scrape_interval: 15s
+      scrape_timeout: 10s
+      evaluation_interval: 15s
+
+    alerting:
+      alertmanagers:
+      - static_configs:
+        - targets: []
+        scheme: http
+        timeout: 10s
+
+    scrape_configs:
+      - job_name: prometheus
```

Comment:
I don't really think it much matters, but do we need prometheus scraping itself in this scenario? Not sure it really adds any overhead or not, but I'd imagine our users would never look at the metrics produced by doing this.

Reply:
agreed, I'll remove it, that's coming from the stock NoSQLBench config

Reply:
Done